torch/_dynamo/skipfiles.py

```diff
@@ -163,6 +163,12 @@ def _module_dir(m: types.ModuleType):
 FBCODE_SKIP_DIRS = {
     "torchrec/distributed",
     "torchrec/fb/distributed",
+    # JaggedTensor and KeyedJaggedTensor are fairly badly behaved, because
+    # they tend to contain extremely long lists of integers that vary from
+    # run to run.  We are interested in tracing models that contain references
+    # to KJTs, but tracing an individual KeyedJaggedTensor is rarely
```

Comment:
Wondering whats the story for export here.

Reply:
for export you aren't going to start the export from jagged_tensor.py anyway, so this heuristic is... not relevant for this (legitimate) question

Reply:
To clarify @anijain2305 - skipfiles are only consulted when we decide where to enter. The export entry point will almost certainly never be a skipfile, and so this should be sound for that.

@ezyang this *might* still cause issues if you are inlining in the middle of export. Inlining checks skipfiles.

Reply:
Oh, that sounds bad, I don't want export to not work haha (not that it works today anyway, but...)